src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodcount/Example4.java

```diff
@@ -0,0 +1,28 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MethodCount">
+      <property name="maxPrivate" value="1"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.sizes.methodcount;
+
+// xdoc section -- start
+class Example4 { // violation, 'Number of private methods is 2 (max allowed is 1)'
+
+  public void publicMethod1() {}
```

Comment:
please copy Example1, name it as Example4 , change config to have `<property name="maxPrivate" value="1"/>` , update comments to match validation.

same for all other examples